tests/manual/tester/src/com/oracle/javafx/tester/ATableViewResizeTester.java

```diff
@@ -0,0 +1,652 @@
+    }
+
+    @Override
+    public void start(Stage stage) throws Exception {
+        contentPane = new BorderPane();
+
+        // selector
+        demoSelector = new ComboBox<>();
+        demoSelector.getItems().addAll(Demo.values());
+        demoSelector.setEditable(false);
+        demoSelector.getSelectionModel().selectedItemProperty().addListener((s,p,c) -> {
+            updatePane();
+        });
+
+        policySelector = new ComboBox<>();
+        policySelector.getItems().addAll(Policy.values());
+        policySelector.setEditable(false);
+        policySelector.getSelectionModel().selectedItemProperty().addListener((s,p,c) -> {
+            updatePane();
+        });
+
+        // layout
+
+        SplitPane split = new SplitPane(contentPane, new BorderPane());
+
+        HBox hb = new HBox(
+            new Label("Data:"),
+            demoSelector,
+            new Label(" Policy:"),
+            policySelector
+        );
+        hb.setSpacing(5);
+
+        BorderPane bp = new BorderPane();
+        bp.setTop(hb);
+        bp.setCenter(split);
+
+        SwingNode swn = new SwingNode();
+        EventQueue.invokeLater(() -> {
+            swingPanel = new SwingPanel();
+            swn.setContent(swingPanel);
+        });
+
+        SplitPane sp = new SplitPane(bp, new BorderPane(swn));
+        sp.setOrientation(Orientation.VERTICAL);
+
+        stage.setScene(new Scene(sp));
+        stage.setWidth(1000);
+        stage.setHeight(500);
+        stage.setTitle("TableView/JTable Resize Tester " + System.getProperty("java.version"));
+        stage.show();
+
+        demoSelector.getSelectionModel().selectFirst();
+        policySelector.getSelectionModel().selectFirst();
+    }
+
+    protected Callback<ResizeFeatures,Boolean> wrap(Callback<ResizeFeatures,Boolean> policy) {
+        return new Callback<ResizeFeatures,Boolean>() {
+            @Override
+            public Boolean call(ResizeFeatures f) {
+                Boolean rv = policy.call(f);
+                int ix = f.getTable().getColumns().indexOf(f.getColumn());
+                System.out.println(
+                    "col=" + (ix < 0 ? f.getColumn() : ix) +
+                    " delta=" + f.getDelta() +
+                    " w=" + f.getTable().getWidth() +
+                    " rv=" + rv
+                );
+                return rv;
+            }
+        };
+    }
+
+    protected String describe(TableColumn c) {
+        StringBuilder sb = new StringBuilder();
+        if(c.getMinWidth() != 10.0) {
+            sb.append("m");
+        }
+        if(c.getPrefWidth() != 80.0) {
+            sb.append("p");
+        }
+        if(c.getMaxWidth() != 5000.0) {
+            sb.append("X");
+        }
+        return sb.toString();
+    }
+
+    protected Callback<ResizeFeatures, Boolean> createPolicy(Policy p) {
+        switch(p) {
+        case AUTO_RESIZE_FLEX_HEAD:
+            return ConstrainedColumnResize.forTable(ResizeMode.AUTO_RESIZE_FLEX_HEAD);
```

Comment:
Use the public API fields, e.g., `TableView.CONSTRAINED_RESIZE_POLICY_FLEX_HEAD`